package.json

```diff
@@ -50,7 +50,9 @@
     "node-mocks-http": "1.2.3",
     "sinon": "1.17.6",
     "typescript": "2.0.10",
-    "typings": "2.0.0"
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
apropos this, looks like v0 was added on march 24: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/node/v0, so we can update here if we want.

might be best to drop support for <4 soon and make it easier on ourselves at this point.